pennylane/operation.py

```diff
@@ -582,21 +582,79 @@ def decompose(self):
             return self.decomposition(wires=self.wires)
         return self.decomposition(*self.parameters, wires=self.wires)
 
+    @staticmethod
+    def compute_diagonalizing_gates(*params, wires, **hyperparams):
+        r"""Defines a partial representation of this operator via
+        its eigendecomposition.
+
+        Given the eigendecomposition `O = U \Sigma U^{\dagger}` where
+        :math:`\Sigma` is a diagonal matrix containing the eigenvalues,
+        the sequence of diagonalizing gates implements the unitary :math:`U`.
+        In other words, the diagonalizing gates rotate the state into the eigenbasis
+        of this operator.
+
+        This is the static version of ``diagonalizing_gates``, which can be called
+        without creating an instance of the class.
+
+        .. note::
+
+            This method gets overwritten by subclasses to define the representation of a particular operator.
+            By default, this method should always take the operator's parameters, wires and hyperparameters as
+            inputs (even if the diagonalizing gates are independent of these values).
+
+            Alternatively, a custom signature can be defined, in which case the `diagonalizing_gates()`
```

Comment:
```suggestion
            Alternatively, a custom signature can be defined, in which case the ``diagonalizing_gates()``
```

Reply:
I am so sorry these still happen to me!! Every time I learnt the lesson, I type in the other framework for a while and unlearn it

Reply:
all good! literally everyone does this, including myself 😆 I remember the Python developers had a ReST survey last year, and the number one feature people asked for was changing the number of backticks needed for code formatting lol